docs/components/concepts/backups.md

```diff
@@ -12,29 +12,47 @@ A Camunda 8 SaaS backup consists of a data backup of Zeebe, Operate, Tasklist, O
 
 With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes, and should not be used for archival of process data.
 
-## Backup and cluster relationship
+## Cluster relationship
 
 Backups are created and managed on a per-cluster basis. It is important to be aware that deleting a cluster will also delete all associated backups.
 
 :::caution
 Exercise caution when deleting clusters to avoid unintended loss of backups.
 :::
 
-Your cluster generation needs to be greater or equal to `8.2.4` to support backups.
+## Programmatic access
+
+The backup operations can be performed programmatically using the Console API. This provides the flexibility to seamlessly integrate backup-related tasks with your existing systems and automation workflows. For detailed information on using the API, refer to the [Console API reference](docs/apis-tools/console-api-reference.md).
+
+## Backup Types
+
+### Manual backup
 
-## Rate limits
+Manual backups are be triggered by the user.
```

Comment:
Let's add a bit more explanation here:
Manual backups refer to the user-initiated process of creating a consistent snapshot of the state of all system components, including Zeebe, Operate, Tasklist,  and Optimize. These backups are managed on a per-cluster basis and are primarily designed for disaster recovery purposes.